shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -647,6 +655,23 @@ - (void)testPropagatePressEventsToViewController2 {
                                              withEvent:[OCMArg isNotNil]]);
 }
 
+- (void)testUpdateSecureTextEntry {
+  NSDictionary* config = self.mutableTemplateCopy;
+  [config setValue:@"YES" forKey:@"obscureText"];
+  [self setClientId:123 configuration:config];
+
+  NSArray<FlutterTextInputView*>* inputFields = self.installedInputViews;
+  FlutterTextInputView* inputView = inputFields[0];
+
+  XCTAssertTrue(inputView.isSecureTextEntry);
+
+  config = self.mutableTemplateCopy;
+  [config setValue:@"NO" forKey:@"obscureText"];
+  [self updateConfig:config];
+
+  XCTAssertFalse(inputView.isSecureTextEntry);
```

Comment:
Can we also test if `[inputView reloadInputViews];` is called?

Reply:
done